src/components/Shared/HeaderAlert.tsx

```diff
@@ -0,0 +1,34 @@
+import Alert from '@mui/material/Alert';
+import AlertTitle from '@mui/material/AlertTitle';
+import { connect } from 'react-redux';
+
+import useAlert from '../../hooks/useAlert';
+import type { AlertType, State } from '../../types/state';
+
+function HeaderAlert(props: HeaderAlertProps) {
+  const { alert } = props;
+  const { useClearAlert } = useAlert();
+  return (
+    <Alert
+      severity={alert.severity}
+      onClose={() => {
+        useClearAlert();
+      }}
+    >
+      {alert.title && <AlertTitle>{alert.title}</AlertTitle>}
+      {alert.message}
```

Comment:
What happens if you have `isAlert === true` but both title and message are empty? Should we display some default message?

Reply:
`title` is the only parameter with a default value, so if you try to call `useSetAlert` without a message, it will throw an error.

I could add a default message though, as there may be a use case in the future where an unexpected error occurs.